gandi/cli/commands/certificate.py

```diff
@@ -192,20 +251,78 @@ def export(gandi, resource, output, force):
               help='Give the DCV method to use to check domain ownership.')
 @pass_gandi
 def create(gandi, csr, private_key, common_name, country, state, city,
-           organisation, branch, duration, package, altnames, dcv_method):
+           organisation, branch, duration, package, flavor, max_altname,
+           altnames, dcv_method):
     """Create a new certificate."""
     if not (csr or common_name):
         gandi.echo('You need a CSR or a CN to create a certificate.')
         return
 
+    if package and (flavor or max_altname):
+       gandi.echo('Please do not use --package at the same time you use '
+                  '--flavor or --max-altname.')
+       return
+
     csr = gandi.certificate.process_csr(common_name, csr, private_key, country,
                                         state, city, organisation, branch)
     if not csr:
         return
 
+    if not common_name:
+        common_name = gandi.certificate.get_common_name(csr)
+
+    if '*' in common_name and altnames and len(altnames) > 1:
+        gandi.echo("You can't have a wildcard with multidomain certificate.")
+        return
+
+    if package:
+        gandi.echo('Using --package is deprecated, please replace it by '
```

Comment:
I would have prefixed this warning with a /!\ or a * to separate this from normal output